source/FreeplayState.hx

```diff
@@ -177,11 +177,11 @@ class FreeplayState extends MusicBeatState
 		add(textBG);
 
 		#if PRELOAD_ALL
-		var leText:String = "Press SPACE to listen to the Song / Press CTRL to open the Gameplay Changers Menu / Press RESET to Reset your Score and Accuracy.";
-		var size:Int = 16;
+		var leText:String = "1.1 is gonna be lit yo! also press space to listen to a song. its a good tip!";
```

Comment:
Even more useless text changes. Not even going to bother with these anymore.

Reply:
I will bother